screens/ScanScreen.js

```diff
@@ -45,15 +45,22 @@ export default function ScanScreen(){
         <View style={styles.container}>
           <View style={styles.barcodebox}>
           <BarCodeScanner
-            onBarCodeScanned={scanned ? undefined : handleBarCodeScanned}
+            onBarCodeScanned={scanned ? false : handleBarCodeScanned}
             style={{height:400, width:400}}
           />
           </View>
           <Text style={styles.maintext}>{text}</Text>
           {scanned && 
+          <View>
             <Pressable style={styles.buttonTapAgain} onPress={() => setScanned(false)}>
-              <Text style={{color: 'white', fontSize:18}}>Scanne à nouveau</Text>
-            </Pressable >}
+              <Text style={{color: 'white', fontSize:18}}>Scanner à nouveau</Text>
+            </Pressable >
+            <Pressable style={styles.addWallet} onPress={() => props.navigation.navigate('BottomNavigator', {screen:'Cartes Fid'})}>
+              <Text style={{color: 'white', fontSize:18}}>Ajouter aux Cartes Fid</Text>
```

Comment:
```suggestion
              <Text style={{color: 'white', fontSize:18}}>Ajouter aux Cartes</Text>
```
Pas génial le "cartes fid"